app/src/hooks/userVendors.ts

```diff
@@ -13,24 +13,18 @@ type UserInfo = {
   name: string;
   email: string;
   mobile?: string;
+  dob?: string;
 };
 
 type Hooks = {
   signup: (userInfo: UserInfo, vendorId: number) => Promise<void>;
-  syncDetail: (
-    name: string,
-    password: string,
-    email: string,
-    dob: string,
-    vendorId: number
-  ) => Promise<void>;
 };
 
 const useVendors = (): Hooks => {
   const api = useApi();
 
   const signup = async (userInfo: UserInfo, vendorId: number) => {
-    const { name, email, mobile } = userInfo;
+    const { name, email, mobile, dob } = userInfo;
```

Comment:
do we need DOB for GPIB?

Reply:
Need yob in gpib

Reply:
@xiangzhe0812 as discussed, lets post full DOB claim